rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1200,25 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _process_contract_info(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: ERC20_INFO_METHODS,
+            contract: EthereumContract,
+    ) -> List[Optional[Union[int, str, bytes]]]:
+        """Decodes information i.e. (decimals, symbol, name) about the token contract.
+        - `decimals` property defaults to 18.
+        - `name` and `symbol` default to None.
+        """
+        decoded_contract_info = []
+        assert len(output) == len(properties), 'output and properties should have the same length'  # noqa: E501
+        for x, method_name in zip(output, properties):
+            if x[0] is True and len(x[1]) != 0:
+                decoded_contract_info.append(contract.decode(x[1], method_name)[0])
+            else:
+                if method_name == 'decimals':
+                    decoded_contract_info.append(DEFAULT_TOKEN_DECIMALS)
+                else:
+                    decoded_contract_info.append(None)
+        return decoded_contract_info
```

Comment:
Make it a bit easier to read

```suggestion
                continue

            # for missing methods, use default decimals for decimals or None for others 
            if method_name == 'decimals':
                decoded_contract_info.append(DEFAULT_TOKEN_DECIMALS)
            else:
                decoded_contract_info.append(None)

        return decoded_contract_info
```